doc/release/release_dev.rst

```diff
@@ -17,22 +17,23 @@ New Features
 - manual segmentation with matplotlib (#2584)
 - hysteresis thresholding in filters (#2665)
 - lookfor function (#2713)
+- montage function (#2626)
 
 
 Improvements
 ------------
 
 
 
-
 API Changes
 -----------
-
-
+- ``skimage.util.montage.montage2d`` is now available as ``skimage.util.montage2d``.
```

Comment:
Because I misread ;)  I thought you meant to write `montage2d` -> `montage`.  My mistake!